docs/data-collection/scanning.md

```diff
@@ -102,15 +103,15 @@
     ???+ quote "Participant of Cohort II"
 
         Hey [NAME], the next block is a bit long, around 30 minutes.
-        
+
         You can close your eyes and even sleep if you wish.
```

Comment:
We may need to change this if we record ET data during DWI

Reply:
Good catch!

Reply:
Yes, no problem!